src/LandIce/problems/LandIce_StokesFOThermoCoupled.hpp

```diff
@@ -489,7 +466,7 @@ constructEnthalpyEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>& fm0,
   p->set<std::string>("Diff Enthalpy Variable Name", "Diff Enth");
   p->set<std::string>("Velocity QP Variable Name", dof_names[0]);
   p->set<std::string>("Velocity Gradient QP Variable Name", dof_names[0] + " Gradient");
-  p->set<std::string>("Vertical Velocity QP Variable Name", "W");
+  p->set<std::string>("Vertical Velocity QP Variable Name", dof_names[1]);
```

Comment:
See comment above. I think this is correct, which makes me thing the above one is wrong.